modules/document_repository/test/TestPlan.md

```diff
@@ -1,5 +1,7 @@
-1. User has access to document repository if they have the "View and upload files in Document Repository” permission or is the super user.
-   [Automation Testing]
+# Document Repository Test Plan
+
+1. User has access to document repository if they have the "View and upload files in Document Repository” or "Delete files in Document Repository" permission.
```

Comment:
Adds the delete permission which on its own should give access to the module.

Reply:
I tested this and just having the "Delete files in Document Repository" does not seem to grant access to the module.

Reply:
Thanks, I opened #6373 as a new issue.